submarine-server/server-submitter/submitter-k8s/src/main/java/org/apache/submarine/server/submitter/k8s/experiment/codelocalizer/SSHGitCodeLocalizer.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.submarine.server.submitter.k8s.experiment.codelocalizer;
+
+import io.kubernetes.client.models.V1PodSpec;
+
+public class SSHGitCodeLocalizer extends GitCodeLocalizer {
+
+  public SSHGitCodeLocalizer(String url) {
+    super(url);
+  }
+
+  @Override
+  public void localize(V1PodSpec podSpec) {
+  }
```

Comment:
@manirajv06 Thanks for your great contribution. Can you mind add TODO comment in here?

Reply:
@jiwq Scope of this PR/JIRA is to provide support http based git sync mode. Added this class now to get clear picture of how interfaces and its impl look like. More over, we will be adding support for ssh very soon. Will create a jira for the same. cc @wangdatan

Reply:
@jiwq Added.